Client/Frontend/Components/ActionFooterView.swift

```diff
@@ -9,39 +9,40 @@ import ComponentLibrary
 struct ActionFooterViewModel {
     let title: String
     let actionTitle: String
-    let a11yTitleIdentifier: String = AccessibilityIdentifiers.ActionFooter.title
-    let a11yActionIdentifier: String = AccessibilityIdentifiers.ActionFooter.primaryAction
+    let a11yTitleIdentifier: String
+    let a11yActionIdentifier: String
+
+    init(title: String,
+         actionTitle: String,
+         a11yTitleIdentifier: String = AccessibilityIdentifiers.ActionFooter.title,
+         a11yActionIdentifier: String = AccessibilityIdentifiers.ActionFooter.primaryAction) {
+        self.title = title
+        self.actionTitle = actionTitle
+        self.a11yTitleIdentifier = a11yTitleIdentifier
+        self.a11yActionIdentifier = a11yActionIdentifier
+    }
 }
 
 final class ActionFooterView: UIView, ThemeApplicable {
```

Comment:
Should `ActionFooterView` be moved to the component library?

Reply:
Yes, it will be moved. We have a task for it in the backlog.